pkg/apis/cr/v1alpha1/repositoryserver_types_test.go

```diff
@@ -18,9 +18,10 @@ import (
 	"testing"
 
 	"github.com/pkg/errors"
-	. "gopkg.in/check.v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/serializer"
+
+	. "gopkg.in/check.v1"
```

Comment:
@julio-lopez i have arranged packages based on this rule .
```
  gci:
    sections:
      - standard
      - default
      - prefix(github.com/kanisterio/kanister)
      - blank
      - dot
```
sure , let's wait for another review from @viveksinghggits

Reply:
In some of the PRs we have treated `_` blank imports as separate category https://github.com/kanisterio/kanister/pull/2500#discussion_r1675602433 and that's the reason I considered that an acceptable change.

@anishbista60, I am assuming that gci is recommending `. "gopkg.in/check.v1"` to be grouped separately just because it's a dot import. 

Now, if we plan to remove the `.` and make it a standard import, we can easily group it with the third party imports but if it's being imported as `.` or `_` I think gci is going to recommend it to group separately.

Reply:
To make things clear I am not recommending that we should change the `.` import to standard import in this PR itself, we can do that later as well. But if we have `.` or `_` blank import, gci would recommend to group that separately.

Reply:
The _"right thing to do"_ here would be to remove the `.` import for `check.v1` and replace all the call sites to include the `check` prefix.

Reply:
> The _"right thing to do"_ here would be to remove the `.` import for `check.v1` and replace all the call sites to include the `check` prefix.

I agree. Should we do that as part of this PR itself.

Reply:
> I agree. Should we do that as part of this PR itself?

- Right now is as good of a time as ever.
- Those changes should be done before enabling the `gci` linter, otherwise this has the risk of falling through the cracks and never getting done.

Reply:
Sounds good. @anishbista60 let's make that change. Please let me know if you have some questions about how to implement this.

Reply:
yes, that looks like the correct order. also, can you please check how `goimports` handles the blank and `.` imports.

Reply:
does it treat them as separate group or not?

Reply:
@viveksinghggits goimports does not treat blank (_) and dot (.) imports as a separate group by default.

Reply:
ok.

Reply:
@viveksinghggits There is are so many places that i am facing error after removing `.`  Should i replace it with `check` every where ?

for example : 

- `c.Assert(err, check.IsNil)` is throwing error . So, it change it to `c.Assert(err, check.IsNil)`
- `c.Assert(snaps,Not(HasLen),0)`   to  `c.Assert(snaps, check.Not(check.HasLen), 0)`

Reply:
Let me make changes to some file and push it . After reviewing , we can move futher.